_posts/2024-07-08-idea-editor.adoc

```diff
@@ -0,0 +1,130 @@
+---
+title: IntelliJ IDEA as a desktop editor for Eclipse Che Cloud Development Environment (CDE)
+layout: post
+author: Artem Zatsarynnyi
+description: >-
+  Integration of Eclipse Che with IntelliJ IDEA Ultimate via JetBrains Gateway.
+categories: []
+keywords: ['Eclipse Che editor', 'IntelliJ IDEA', 'JetBrains Gateway']
+slug: /@artem/idea-editor
+---
+
+In this blog post, we'll look at the integration of Eclipse Che with IntelliJ IDEA.
+
+It brings the local development experience to the remote development allowing the user to use their familiar IDE when working with the Eclipse Che CDE.
+
+image::/assets/img/idea-editor/main.png[Eclipse Che and JetBrains Gateway integration]
+
+== Why another integration with JetBrains IDEs?
+
+In addition to VS Code, Eclipse Che also https://github.com/che-incubator/jetbrains-editor-images/[provides] the in-browser variant of IntelliJ IDEA Community Edition as one of the editor options to use when working with your CDE. It's based on the JetBrains Projector.
+
+> Projector is a self-hosted technology that runs IntelliJ-based IDEs and Swing-based apps on the server, allowing you to access them from anywhere using browsers and native apps.
+> -- From the https://blog.jetbrains.com/blog/2021/03/11/projector-is-out/[JetBrains blog].
+
+JetBrains https://lp.jetbrains.com/projector/[suspended] the development and support of the Projector as a standalone project and introduced a new Projector-based product - https://www.jetbrains.com/remote-development/gateway/[JetBrains Gateway]. It allows the use of the desktop IDE instead of the in-browser web client. So, we integrated its support to Eclipse Che.
```

Comment:
```suggestion
JetBrains https://lp.jetbrains.com/projector/[suspended] the development and support of the Projector as a standalone project and introduced a new product - https://www.jetbrains.com/remote-development/gateway/[JetBrains Gateway]. It allows the use of the desktop IDE instead of the in-browser web client, and we integrated its support into Eclipse Che.
```